data/com.mattjakeman.ExtensionManager.desktop.in.in

```diff
@@ -8,5 +8,6 @@ Categories=GTK;Utility
 StartupNotify=true
 MimeType=x-scheme-handler/gnome-extensions;
 Comment=Manage GNOME Shell Extensions
-# Translators: Do not translate
+# Translators: Search terms to find this application. Do NOT translate or localize the semicolons. The list MUST also end with a semicolon.
+# Make sure to leave the original strings and add your translations afterwords, so that users can search both in English and your language.
```

Comment:
```suggestion
# Make sure to leave the original strings and add your translations afterwards, so that users can search both in English and your language.
```

Reply:
Ah...😵 Thank you for pointing out.